tests/integration/transaction_freeze_e2e_test.py

```
@@ -0,0 +1,146 @@
+import pytest
+
+from hiero_sdk_python.account.account_id import AccountId
+from hiero_sdk_python.client.client import Client
+from hiero_sdk_python.consensus.topic_create_transaction import TopicCreateTransaction
+from hiero_sdk_python.response_code import ResponseCode
+from hiero_sdk_python.transaction.transaction import Transaction
+from hiero_sdk_python.transaction.transaction_id import TransactionId
+from tests.integration.utils import env
+
+@pytest.mark.integration
+def test_transaction_executes_successfully(env):
```

Ruff (0.14.10)

Redefinition of unused `env` from line 9

(F811)